money.gemspec

```diff
@@ -14,11 +15,10 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency("bundler", ">= 1.5")
   s.add_development_dependency("simplecov", ">= 0")
-  s.add_development_dependency("rails", "~> 5.0")
+  s.add_development_dependency("rails", "~> 6.0")
```

Comment:
Need to get rid of the 2.3, 2.4  ruby version from the matrix CI

Reply:
And change the `required_ruby_version`